hadoop-hdfs-project/hadoop-hdfs-client/src/main/java/org/apache/hadoop/hdfs/web/WebHdfsFileSystem.java

```diff
@@ -1459,6 +1460,19 @@ SnapshotDiffReport decodeResponse(Map<?, ?> json) {
     }.run();
   }
 
+  public SnapshotStatus[] getSnapshotList(final Path snapshotDir)
```

Comment:
In DistributedFileSystem, it is called
`  public SnapshotStatus[] getSnapshotListing(Path snapshotRoot)`
Let's use the same name?